cfme/cloud/keypairs.py

```diff
@@ -89,7 +87,6 @@ def is_displayed(self):
 
     toolbar = View.nested(KeyPairToolbar)
     entities = View.nested(KeyPairEntities)
```

Comment:
please use include  here

Reply:
I ended up removing empty KeyPairEntities and used View.include for VMEntities directly here.